src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,35 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+
+  grid-template-columns: repeat(4, $card-width);
+
+  gap: 48px;
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, $card-width);
+  }
```

Comment:
It's an option, I'm just curious, is there a particular reason why you decided to take desktop first approach here?:thinking:

Reply:
there is no specific reason, I think it would be better to use mobile first approach in the future